### ISSUE TITLE
install: cannot stat 'libopenblas_haswellp-r0.2.18.so': No such file or directory

### ISSUE THREAD

**Issue by svm-zhang** [PacificBiosciences/pitchfork#313]
Hello,

I was trying to `make pbalign` using pitchfork on a docker container with Ubuntu 15.04 as base image. The process failed at openBlas step and the error was as shown in the title. Could you please elaborate on this error and how could I fix it? Any help would be appreciated!

Simo

**Comment by mhsieh** [comment]
We might need some more error log prior to that.

**Comment by mhsieh** [comment]
My test under a ubuntu 1504 vagrant box works just fine. So I guess I will offer you a quick fix first. 

Untar this tarball https://www.dropbox.com/s/tzssy09jdm2z85k/OpenBLAS.tar?dl=0 into your PREFIX (if you didn't set it, it's under pitchfork/deployment). Let's first unblock your progress, and then figure out the problem.

Yes, use it with your own risk :-D

**Comment by svm-zhang** [comment]
Hello,

Thanks for the quick reply! Please see the attached file.

[pbalign.install.stderr.txt](https://github.com/PacificBiosciences/pitchfork/files/568674/pbalign.install.stderr.txt)

BTW, `make blasr` finishes without any error.

Let me know if you need anything from me, and thanks again.

Simo

**Comment by svm-zhang** [comment]
Untar the tarball into deployment folder fixed the error. :)

I saw in the tarball you have some header and the missing lib file. Is this lib file not shipped with default Ubuntu 15.04 box? I also tried to install libopenblash-dev but did not work. Or do I need to install OpenBlas myself and copy the header and lib files to deployment folder?

**Comment by mhsieh** [comment]
The design was to use as little system file as possible, I will work on the option to allow using system libopenblas-dev instead of building it.

**Comment by svm-zhang** [comment]
I am wondering is the system libopenblas-dev has the missing lib file?

**Comment by mhsieh** [comment]
I will be back to you on that: but next time you probably could do this `make openblas LIBCORE=nehalem CORE=NEHALEM` before `make pbalign`

**Comment by mhsieh** [comment]
Add this to your settings.mk if you want the system openblas (make sure to do `apt-get install libopenblas-dev` first)
```
HAVE_OPENBLAS = /usr
```

**Comment by mhsieh** [comment]
On ubuntu-15.04, it might be better if you use system python. The setting for reuse the system python is:
```
USE_PYTHON =
```
(Make sure you do `apt-get install python-dev` first)

**Comment by mhsieh** [comment]
Please reopen this if you can't reuse the system openblas lib.

**Comment by svm-zhang** [comment]
hello @mhsieh , let me try to rebuild it with all what you have told me and get back to you. Thanks!

**Comment by svm-zhang** [comment]
Hello @mhsieh, I have tried to rebuild pbalign from scratch using both system openblas and python, it did pass the openblas error I had before, but it complained about not find lpbbam. What I did was adding a symbolic link of libpbbam.so inside the deployment/lib folder, and solved the error. The rest installation ran without any problems.

**Comment by mjhsieh** [comment]
Not sure why you need that kind of fix, but please find open a new issue if you found any problem in the future. 🙏
